### PR TITLE
Made Alolan Marowak be affected by Thick Club

### PR DESCRIPTION
**Pull request by LOuroboros**
## Description
The Thick Club is supposed to affect Alolan Marowak, but it wasn't in the BE so I quickly address that with this PR.

## **Discord contact info**
Lunos#4026

### CODE REVIEW

**Comment by ultima-soul** [comment]
I wonder if we should wait to incorporate #1316 so you'd utilize something like it for checking Marowak species. 

Might be more flexible in general to do all species checks like this so that if people say add a regional form or any kind of form for these mons, it would automatically check for them instead of having to manually add them or change it to using this kind of check for Cubone on their own.

**Comment by LOuroboros** [comment]
> 
> 
> I wonder if we should wait to incorporate #1316 so you'd utilize something like it for checking Marowak species.
> 
> Might be more flexible in general to do all species checks like this so that if people say add a regional form or any kind of form for these mons, it would automatically check for them instead of having to manually add them or change it to using this kind of check for Cubone on their own.

Uh, sure, that sounds good to me. I actually forgot that PR was a thing honestly.

One thing though, it seems to be missing the alolan forms including Marowak's. Unless maybe you plan to incorporate that into the PE too? :eyes:

**Comment by ExpoSeed** [comment]
Yes the alolan form should be listed in #1315, so then this check changes to "if the base species is marowak"

**Comment by LOuroboros** [comment]
Ready for review :eyes:

File: src/battle_util.c [hunk 1]
```diff
@@ -7116,7 +7116,9 @@ static u32 CalcAttackStat(u16 move, u8 battlerAtk, u8 battlerDef, u8 moveType, b
     switch (GetBattlerHoldEffect(battlerAtk, TRUE))
     {
     case HOLD_EFFECT_THICK_CLUB:
-        if ((gBattleMons[battlerAtk].species == SPECIES_CUBONE || gBattleMons[battlerAtk].species == SPECIES_MAROWAK) && IS_MOVE_PHYSICAL(move))
+        if ((GET_BASE_SPECIES_ID(battlerAtk) == SPECIES_CUBONE
+         || GET_BASE_SPECIES_ID(battlerAtk) == SPECIES_MAROWAK)
```
Comment (ultima-soul) [inline src/battle_util.c:7120]:
```suggestion
        if ((GET_BASE_SPECIES_ID(gBattleMons[battlerAtk].species) == SPECIES_CUBONE
         || GET_BASE_SPECIES_ID(gBattleMons[battlerAtk].species) == SPECIES_MAROWAK)
```

**Review by ultima-soul — CHANGES_REQUESTED** [review]

**Comment by LOuroboros** [comment]
Ready for re-review 👀

**Review by ultima-soul — APPROVED** [review]

**Review by ghoulslash — APPROVED** [review]

**Pull request merged**